src/kili/mutations/plugins/queries.py

```diff
@@ -0,0 +1,31 @@
+"""
+Queries of plugins mutations
+"""
+
+GQL_ACTIVATE_PLUGIN_ON_PROJECT = """
+mutation(
+  $pluginName: String!
+  $projectId: ID!
+  ) {
+  data: activatePluginOnProject(
+    where: {
+      pluginName: $pluginName
+      projectId: $projectId
+    }
+  )
+}
+"""
+
+GQL_DEACTIVATE_PLUGIN_ON_PROJECT = """
+mutation(
+  $pluginName: String!
+  $projectId: ID!
+  ) {
+  data: deactivatePluginOnProject(
+    where: {
+      pluginName: $pluginName
+      projectId: $projectId
+    }
+  )
+}
```

Comment:
duplicates with the file in graphql/operations/plugins/mutations ?

Reply:
same, it was for the POC. I'll remove it

Reply:
No it's not even needed for the POC. I just forgot to delete it